test/instrumentation/modules/next/next.test.js

```diff
@@ -774,7 +775,7 @@ tape.test('-- dev server tests --', suite => {
       waitForServerReady(t, waitErr => {
         if (waitErr) {
           t.fail(`error waiting for Next.js server to be ready: ${waitErr.message}`)
-          nextServerProc.kill('SIGKILL')
+          treekill(nextServerProc.pid, 'SIGKILL')
```

Comment:
REVIEW NOTE: The `treekill` usage was added because on macOS I was running into a failure in `TAV=next npm run test:tav` when the *second* `next` version was run. The used port (port 3000) was still busy when the `next start` server was started for the second run. It was as if the process for the `next dev` server from the previously tested version was still running.  The `treekill` usage proved useful for a similar case of a process *tree* in the azure-functions tests.